pkg/kademlia/endpoint.go

```diff
@@ -106,7 +106,7 @@ func (endpoint *Endpoint) RequestInfo(ctx context.Context, req *pb.InfoRequest)
 
 	if self.Type == pb.NodeType_STORAGE {
 		if endpoint.trust == nil {
-			return nil, status.Error(codes.Internal, "missing trust verifier")
+			return nil, status.Error(codes.Internal, "missing trust")
```

Comment:
I would say to add the log Error message here, but that work isn't part of this branch and anyway this package is going away soon.

Reply:
Yeah, I think this check shouldn't be even needed and the constructor should panic/fail.